### PR TITLE
Update udigest

### PR DESCRIPTION
**Pull request by survived**
Changelog pipeline is failing, but there's nothing to add to changelog in this case

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -11,3 +11,7 @@ members = [
 exclude = [
   "wasm/nostd",
 ]
+
+[patch.crates-io.udigest]
+git = "https://github.com/dfns/udigest"
+branch = "small-improvements"
```
Comment (survived) [inline Cargo.toml]: to be deleted

**Review by maurges — COMMENTED** [review]
Fun, so the changelog script doesn't live well with rc versions

**Comment by survived** [comment]
Yeah, but that doesn't happen very often that we have rc versions, so it's easier to ignore it than to update the script

**Review by maurges — DISMISSED** [review]

**Comment by survived** [comment]
There's another release of `udigest` upcoming... I think it's better to update everything in bulk when udigest `v0.2` is out

**Comment by survived** [comment]
@maurges updated deps so there's no `rc` versions anymore

**Review by maurges — APPROVED** [review]

**Pull request merged**